### PR TITLE
Adds notice-span to visible_messages with no spans

### PR DESCRIPTION
**Pull request by kingofkosmos**
##  About The Pull Request

Adds 'notice' span class to all visible_messages which had no span class, making all those black messages blue.

## Why It's Good For The Game

This should help differentiate action-messages from talking-messages in the chat. More actions will be blue, thus black talking-messages should pop out more.

### CODE REVIEW

**Comment by kingofkosmos** [comment]
Alright, this is ready for reviewing.

**Comment by kingofkosmos** [comment]
Compiles fine on my end.

File: code/modules/goonchat/browserassets/css/browserOutput.css [hunk 1]
```diff
@@ -253,7 +253,7 @@ a.popt {text-decoration: none;}
 .bold, .name, .prefix, .ooc, .looc, .adminooc, .admin, .medal, .yell {font-weight: bold;}
 
 /* ADD HERE FOR ITALIC */
-.italic, .italics,  .emote {font-style: italic;}
+.italic, .italics,  .emote {color: #6685f5;	font-style: italic;}
```
Comment (actioninja) [inline code/modules/goonchat/browserassets/css/browserOutput.css]: don't do this.
italics class is for italics.

File: code/modules/goonchat/browserassets/css/browserOutput_white.css [hunk 2]
```diff
@@ -250,7 +250,7 @@ a.popt {text-decoration: none;}
 .bold, .name, .prefix, .ooc, .looc, .adminooc, .admin, .medal, .yell {font-weight: bold;}
 
 /* ADD HERE FOR ITALIC */
-.italic, .italics,  .emote {font-style: italic;}
+.italic, .italics,  .emote {color: #000099;	font-style: italic;}
```
Comment (actioninja) [inline code/modules/goonchat/browserassets/css/browserOutput_white.css]: don't do this.
italics class is for italics.

**Review by actioninja — CHANGES_REQUESTED** [review]

File: code/modules/goonchat/browserassets/css/browserOutput.css (hunk 1 above)
Comment (kingofkosmos) [reply code/modules/goonchat/browserassets/css/browserOutput.css]: Alright, I thought it wasn't used anywhere else but heard messages. I'll add a new hearing -span class in another PR and change all heard messages to that.

**Comment by optimumtact** [comment]
There will be an unclosed span I know it

**Comment by MrDoomBringer** [comment]
https://github.com/tgstation/tgstation/pull/30173/files

https://github.com/tgstation/tgstation/pull/46044/files#diff-12e81e9f891a99248867bd1d8a818e7bR27

oranges giveth and oranges mergeth away

**Pull request merged**